lxml-stubs/etree.pyi

```diff
@@ -532,8 +532,8 @@ class XPath:
         self,
         path: _AnyStr,
         *,
-        namespaces: Optional[_AnyStr] = ...,
-        extensions: Optional[_AnyStr] = ...,
+        namespaces: Optional[_DictAnyStr] = ...,
+        extensions: Any = ...,
```

Comment:
Probably ok to spell it that way (it's not very important), but it's really
`[ {(ns, name):function} ]`